src/main/resources/upgrade-files/post-upgrade/sql/UAF-4199Remove_Discontinued_Outbound_Messaging_MOD.sql

```diff
@@ -0,0 +1,20 @@
+--=======================================================================================================================================
```

Comment:
Does this need to go into kfsdbupgrade.properties:69, appended to the "files-post-upgrade" prop?